data.sql

```diff
@@ -6,3 +6,16 @@ VALUES
     ('Gabumon', '2018-11-15', 8, true, 2),
     ('Pikachu', '2021-01-07', 15.04, false, 1),
     ('Devimon', '2017-05-12', 11, true, 5);
+
+
+
+-- Second day 
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES
+    ('Charmander', '2020-02-08', -11, false, 0, 'Fire'),
+    ('Plantmon', '2021-11-15', -5.7, true, 2, 'Grass'),
+    ('Squirtle', '1993-04-02', -12.13, false, 3, 'Water'),
+    ('Angemon', '2005-06-12', -45, true, 1, 'Angel'),
+    ('Boarmon', '2005-06-07', 20.4, true, 7, 'Mammal'),
+    ('Blossom', '1998-10-13', 17, true, 3, 'Plant'),
+    ('Ditto', '2022-05-14', 22, true, 4, 'Unknown');
```

Comment:
- [ ] **Good job so far, 👍** However, it would be better to add data as it is in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Insert%20the%20following%20data).  

please note that data in the species column should be added Inside a transaction to update the `animals` table.🙏